packages/plugin-timeclock-api/src/graphql/resolvers/mutations.ts

```diff
@@ -172,16 +172,18 @@ const timeclockMutations = {
   },
   absenceTypeAdd(
     _root,
-    { name, explRequired, attachRequired },
+    { name, explRequired, attachRequired, shiftRequest },
     { models }: IContext
   ) {
     const explanationReqd: boolean = explRequired;
     const attachReqd: boolean = attachRequired;
+    const isShiftRequest: boolean = shiftRequest;
```

Comment:
These variables look unnecessary. Just pass the parameters directly to the createAbsenceType method.